spec/factories/card.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+
+  factory :card do
+    original_text "house"
+    translated_text "дом"
+    transcription "[haʊs]"
+
+    trait :expired do
+      review_date -3.day.since
```

Comment:
Ambiguous negative number operator. Parenthesize the method arguments if it's surely a negative number operator, or add a whitespace to the right of the `-` if it should be a subtraction.